man/ukify.xml

```diff
@@ -369,6 +370,27 @@
         </varlistentry>
       </variablelist>
     </refsect2>
+
+    <refsect2>
+      <title>[Addon:<replaceable>NAME</replaceable>] section</title>
+
+      <para>These options apply only when building PE addons.</para>
```

Comment:
hmm, why only to addon? sbat is also included in the UKI, no?

Reply:
that comes from the stub, it's handled separately

Reply:
so i would tone this down a bit.

i.e. say that *currently* this only applies to PE add-ons, not to UKIs. Because as I see it we should sooner or later allow supplying SBAT data for UKIs too when building them here, and then merge the data from the stub, from the embedded kernel and what is supplied here into one sbat blob.

Reply:
I've asked the sbat folks what's the guidance on granularity here